theseus_gui/src/components/RowDisplay.vue

```diff
@@ -0,0 +1,202 @@
+  else if (shouldRenderTrending) trendingRow.value.scrollLeft += 100
+}
+</script>
+
+<template>
+  <div class="row">
+    <div class="header">
+      <p>{{ props.label }}</p>
+      <hr aria-hidden="true" />
+      <div v-if="allowPagination" class="pagination">
+        <ChevronLeftIcon @click="handleLeftPage" />
+        <ChevronRightIcon @click="handleRightPage" />
+      </div>
+    </div>
+    <section ref="modsRow" class="mods" v-if="shouldRenderNormalInstances">
+      <Instance
+        v-for="instance in props.instances"
+        :key="instance.id"
+        display="gallery"
+        :instance="instance"
+      />
+    </section>
+    <section ref="newsRow" class="news" v-else-if="shouldRenderNews">
+      <News v-for="news in props.news" :key="news.id" :news="news" />
+    </section>
+    <section ref="trendingRow" class="trending" v-else-if="shouldRenderTrending">
+      <Instance
+        v-for="instance in props.instances"
+        :key="instance.id"
+        display="card"
+        :instance="instance"
+      />
+    </section>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.row {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  padding: 1rem;
+
+  &:nth-child(odd) {
+    background: var(--color-raised-bg);
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    align-items: inherit;
+    width: 100%;
+    margin-bottom: 1rem;
+    gap: 1rem;
+
+    p {
+      font-size: 0.85rem;
+      white-space: nowrap;
+    }
+
+    hr {
+      background: var(--color-base);
+      height: 1px;
+      width: 100%;
+      border: none;
+    }
+
+    .pagination {
+      display: inherit;
+      align-items: inherit;
+
+      svg {
+        cursor: pointer;
+        margin-right: 0.5rem;
+        transition: all ease-in-out 0.1s;
+
+        &:hover {
+          filter: brightness(150%);
+        }
+      }
+    }
+  }
+
+  section {
+    display: flex;
+    align-items: inherit;
+    transition: all ease-in-out 0.4s;
+    gap: 1rem;
+  }
+
+  .mods {
+    width: 100%;
+    margin: auto;
+    transition: all ease-in-out 0.4s;
+    scroll-behavior: smooth;
+    overflow-x: scroll;
+    overflow-y: hidden;
+
+    &::-webkit-scrollbar {
+      width: 0px;
+      background: transparent;
+    }
+  }
+
+  .news {
+    margin: auto;
+    width: 100%;
+    scroll-behavior: smooth;
+    overflow-x: scroll;
+    overflow-y: hidden;
+
+    &::-webkit-scrollbar {
+      width: 0px;
+      background: transparent;
+    }
+  }
+
+  .trending {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    gap: 1rem;
+    margin-right: auto;
+    margin-top: 0.8rem;
+    scroll-behavior: smooth;
+
+    overflow-x: scroll;
+    overflow-y: hidden;
+
+    &::-webkit-scrollbar {
+      width: 0px;
+      background: transparent;
+    }
+  }
+}
+
+.dark-mode {
+  .row {
+    &:nth-child(even) {
+      background-color: rgba(22, 24, 28, 0.3);
+    }
+  }
+}
```

Comment:
This color is correct but it looks too dark because the base "background" color of the launcher is set incorrectly. It should be the grayish color in the design.

Reply:
Set colors as we discussed.